biosdk-services/src/main/resources/application.properties

```diff
@@ -1,16 +1,31 @@
 server.port=9099
-spring.cloud.config.enabled=false
+#spring.cloud.config.enabled=false
```

Comment:
Since we have disabled this, are we connecting to config server in 1.2.0.1 ? If yes, lets not keep the default value of mosip.biosdk.log-request-response-enabled as true

Reply:
No Vishwa, we are not connecting to config server yet. So I will put it back.